### DISCUSSION TITLE
Deploying multiple production releases from different branches in the same repository.

### DISCUSSION THREAD

**Discussion by mostwise** [vercel/vercel#11673] [Help]
### Question

I have wanted to deploy [a blog](https://blog.shxrk.dev) along with my main website, but my blog is always a preview and only specific people can access it. I would like to have two production releases.

One on [shxrk.dev](https://shxrk.dev)
And another on [blog.shxrk.dev](https://blog.shxrk.dev)

Like I said earlier though, one becomes a preview and I want both of them to be production.

**Comment by versecafe** [comment]
You can only have one production deployment per project, you can always deploy something to two different projects or together to two domains in one by either assigning two domains or using a middleware rewrite to rewrite `/blog` to `blog.*shxrk.dev`
